examples/CosmosDB/CosmosDB.Tests.ps1

```diff
@@ -246,18 +246,6 @@ Describe 'Cosmos DB Gremlin Database' {
     Confirm-AzBPCosmosDBGremlinDatabase @params | Should -Not -BeSuccessful
   }
 
-  It "Should contain a Cosmos DB Gremlin Database named $gremlinDatabaseName in $location" {
```

Comment:
I left a comment on Ozi's PR too, but I noticed we aren't specifying a location parameter in the bicep files for the database accounts, which might be why this is failing?

Reply:
Oo interesting catch. Each of the databases is dependent on the "parent" Cosmos Account, so I assumed it would use the Cosmos Account's location for the database location. Let me try adding the `location` key in the bicep files though!

Reply:
Hm alright just tested it. Added `location` key in bicep files and redeployed from scratch. During the tests, I inspect what Az PowerShell is returning and location is still null :(
![image](https://user-images.githubusercontent.com/15195985/230481583-31202e64-b062-4edd-a127-5e8726b3c966.png)